robber/expect.py

```diff
@@ -19,8 +19,9 @@ class expect:
     matchers = {}
     message = None
 
-    def __init__(self, object):
-        self.object = object
+    def __init__(self, obj):
+        self.obj = obj
+        self.is_negative = False
```

Comment:
@hieueastagile well, you can take a look at the fourth meaning in OALD: http://www.oxfordlearnersdictionaries.com/definition/english/negative_1?q=negative. 
It literally means "containing a word such as ‘no’, ‘not’, ‘never’, etc.". 
Another source is Google Trends: https://trends.google.com/trends/explore?q=negative,negated
You can see that "negative" is more widely used, while negated is really limited.